nebula/ui/components/MZFlickable.qml

```diff
@@ -14,6 +14,8 @@ Flickable {
     property var flickContentHeight
     property bool contentExceedsHeight: height < contentHeight
     property bool hideScollBarOnStackTransition: false
+    property bool navbarVisible: navbar.visible
```

Comment:
Is this property alias required although navbar is a global component?

Reply:
this property is used to tell the flickable whether or not to consider the navbar when calculating its `contentHeight` (eg do we need to be able to scroll more so that no content is stuck behind the navbar). Most of the time this is based on whether the navbar is actually visible or not, but in the case of these sheets that are overlayed on top of the screen, the navbar is still `visible == true`, but it is behind the sheet so does not need to be considered.